### PR TITLE
Make CLI callable from target package.

### PR DESCRIPTION
**Pull request by fbinz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +75,11 @@ METADATA = PackageMetadata(
 )
 ```
  
+* Install the `ambient_package_update` package
+  ```
+  # ideally in a virtual environment
+  pip instal ambient_package_update
```
Comment (GitRon) [inline README.md:81]: missing "l"

Comment (GitRon) [reply README.md:81]: and the package is installed with "-" instead of "_"

Comment (fbinz) [reply README.md]: Done :thumbsup:

File: README.md [hunk 2]
```diff
@@ -3,46 +3,40 @@
...
 This package will render all required configuration and installation files for your target package.
 
 Typical use-cases:
-* A new Python or Django version was release
-* A Python or Django version was deprecated
-* You want to update the Sphinx documentation builder
-* You want to update the linter versions
-* You want to add the third-party dependencies
+
+- A new Python or Django version was release
```
Comment (GitRon) [inline README.md:13]:
```suggestion
- A new Python or Django version was released
```

File: ambient_package_update/templates/pyproject.toml.tpl [hunk 3]
```diff
@@ -54,6 +54,8 @@ name = "{{ package_name }}"
 
 
 [tool.black]
+# use force-exclude, so that black also applies exclude when run using pre-commit: https://github.com/psf/black/issues/395
+force-exclude = '''.*/migrations/.*'''
```
Comment (GitRon) [inline ambient_package_update/templates/pyproject.toml.tpl:58]: neat 👍

File: ambient_package_update/cli.py [hunk 4]
```diff
@@ -107,20 +110,20 @@ def build_docs(package_name: str):
...
 
-    package_data = get_metadata(package_name)
-    dependency_list = package_data['dependencies']
+    package_data = get_metadata()
+    dependency_list = package_data.dependencies
 
-    if package_data['optional_dependencies']:
-        for _, opt_dependency in package_data['optional_dependencies'].items():
+    if package_data.optional_dependencies:
+        for _, opt_dependency in package_data.optional_dependencies.items():
```
[the remark this answers is not in the document]
Comment (fbinz) [reply ambient_package_update/cli.py:120]: It always was, you just forgot to update it in the run_tests function.

**Review by GitRon — APPROVED** [review]

**Pull request merged**